soccer/Robot.hpp

```diff
@@ -530,6 +532,9 @@ class OurRobot : public Robot {
     friend class MotionControl;
 
 private:
+    mutable RJ::Time _lastBallSense;
+    const long _lostBallDuration = 200000000;
```

Comment:
I feel like there's a way to express this in seconds, although I don't know how the time library works too well. 

Try RJ::seconds(0.2)? I'll try to figure it out if it's not clear....